index.js

```diff
@@ -6,6 +6,7 @@ const fs = require('fs');
 const path = require('path');
 const lodash = require('lodash');
 const mediumImport = require('./lib/mediumImport');
+const moment = require('moment');
```

Comment:
hmmm, do we really need to pull in moment for that?
I would rather have something more lightweight, like `date-fns` or something?